src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/CoursesIntegrationTest.kt

```diff
@@ -189,6 +186,8 @@ class CoursesIntegrationTest
       .bodyValue(CsvTestData.prerequisitesCsvText)
       .exchange()
       .expectStatus().is2xxSuccessful
+      .expectBody()
```

Comment:
Ideally this would come in its own commit, or one that refactors things - unless I'm misunderstanding, is there a reason it's been added with the "Provide feedback on unmatched offerings and prerequisites" commit?

Reply:
Each commit builds on the previous one.  I could not have created a separate branch from main for this commit because it depends upon all the commits leading up to this one.
It might help if every commit were considered a release in its own right, each worthy of its own PR and merge to main. This, after all is what continuous integration means.  If you think that all the previous commits are OK then all of them are going to be merged to main. I don't think it makes any difference if you do them one at a time or all at once.  It would certainly be simpler to merge (or ideally rebase) the whole lot on to main in one go and the outcome is the same.

Reply:
I think there might be some crossed wires here - I'm referring to "commit" rather than "branch" - this would be a separate commit, on the same branch.

Reply:
I'll accept updating the commit message here with a bit more detail as to why these tests have been updated, rather than splitting it out into a new commit to help get this over the line.

Reply:
Ah.  Added .expectBody().jsonPath(... because prior to this commit there was no body.  Now there is, and because a number of the prerequisites fail to match a course by name the response body array has 141 entries, one for each mismatch. Line 199 asserts that 94 prerequisites were matched to courses as before.